javascript/javascript3/week1/giphy.js

```diff
@@ -0,0 +1,36 @@
+
+    const divTag=document.querySelector(".container")
+    const btn=document.getElementById("btn")
+    const reSet=document.getElementById("reSet")
+    const getData=()=>{
+     let queryCatagori=search.value;
+      if(!queryCatagori)
+    {
+        alert("please enter the in  search ")
+    }
+
+    let apiUrl=`http://api.giphy.com/v1/gifs/search?&api_key=XxWJEby7Gz2VL9lyDL4lUb9VzLt4sN8G&q=${queryCatagori}`;
+    
+  fetch(apiUrl)
+            .then(res=>res.json())
+            .then(ipData=>{
```

Comment:
Try to rewrite this part (below) as an independent function. Ideally functions should only have one responsibility. If it's fetchData, it should fetch data only. Then you can use a callback to render data.